tests/default_vars.sh

```diff
@@ -674,14 +674,11 @@ export cap_dbug_flag="0"
 export use_coldstart="false"
 export use_mommesh="true"
 export RUNTYPE='startup'
+export CICERUNTYPE='initial'
+export eps_imesh='1.0e-1'
 
 # FV3 defaults
-# to use new oro and ics created from 1deg ocean mask on c96 tiles
-# set frac_grid=.F. but FRAC_GRID_INPUT=.T.
-# to repro existing tests set both frac_grid and FRAC_GRID_INPUT to .F.
-# to run frac_grid, set both frac_grid and FRAC_GRID_INPUTs to .T.
 export FRAC_GRID='.T.'
-export FRAC_GRID_INPUT='.T.'
```

Comment:
This line is removed, does this mean by default we have FRAC_GRID_INPUT?

Reply:
yes, the input is the same between frac grid and non-frac grid; we can switch with the namelist setting. This setting was back when we had a mix of input, some of which was only valid for the frac grid.